Sources/MuxPlayerSwift/Fairplay/FairplaySessionManager.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  FairplaySessionManager.swift
+//
+//
+//  Created by Emily Dixon on 4/19/24.
+//
+
+import Foundation
+import AVFoundation
+
+class FairplaySessionManager {
```

Comment:
Nit: adjust capitalization for consistency, e.g. to match `AVContentKeySystem. fairPlayStreaming`